web/src/lib/components/admin-page/settings/setting-input-field.svelte

```diff
@@ -39,8 +40,14 @@
 			</div>
 		{/if}
 	</div>
+
+	<div class={`flex place-items-center gap-1`}>
+		<label class={`immich-form-label text-xs`} for="{label}-desc">{desc}</label>
+	</div>
```

Comment:
Also for `SettingSelect` component

```suggestion
	{#if desc}
		<p class="immich-form-label text-xs pb-2" id="{name}-desc">
			{desc}
		</p>
	{/if}
```